pkgs/tools/inputmethods/fcitx5/with-addons.nix

```diff
@@ -1,17 +1,20 @@
-{ symlinkJoin, makeWrapper, fcitx5, fcitx5-lua, fcitx5-configtool, fcitx5-qt, fcitx5-gtk, addons ? [] }:
+{ lib, symlinkJoin, makeWrapper, fcitx5, fcitx5-configtool, fcitx5-qt, fcitx5-gtk, addons ? [ ] }:
+
+with lib;
 
 symlinkJoin {
   name = "fcitx5-with-addons-${fcitx5.version}";
 
-  paths = [ fcitx5 fcitx5-configtool fcitx5-lua fcitx5-qt fcitx5-gtk ] ++ addons;
+  paths = [ fcitx5 fcitx5-configtool fcitx5-qt fcitx5-gtk ] ++ addons;
 
   nativeBuildInputs = [ makeWrapper ];
 
   postBuild = ''
     wrapProgram $out/bin/fcitx5 \
       --prefix FCITX_ADDON_DIRS : "$out/lib/fcitx5" \
       --suffix XDG_DATA_DIRS : "$out/share" \
-      --suffix PATH : "$out/bin"
+      --suffix PATH : "$out/bin" \
+      --suffix LD_LIBRARY_PATH : ${makeLibraryPath (flatten (map (x: x.passthru.extraLdLibraries) addons))}
```

Comment:
You should check whether passthru and extraLdLibraries existed for an addon here.

By the way, fcitx5-lua can work without error now.